api/rpc/account/account.go

```diff
@@ -215,6 +216,7 @@ func (s *Server) GetUser(ctx context.Context, in *GetUserRequest) (*GetUserReply
 }
 
 // ListUsers implements account.ListUsers
+// nolint : dupl
```

Comment:
I tried setting `--dup-threshold` in the Makefile to a different value. It seems to work.